dietpi/dietpi-software

```diff
@@ -8296,7 +8303,77 @@ _EOF_
 			G_CONFIG_INJECT 'SNAPCLIENT_OPTS[[:blank:]=]' "SNAPCLIENT_OPTS=\"-h $snapcast_server_ip -p $snapcast_server_port\"" /etc/default/snapclient
 
 		fi
+		
+		software_id=198 # File Browser
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			# ARMv6		
+			local arch='armv6'
+
+			# ARMv7
+			if (( $G_HW_ARCH == 2 )); then
+				arch='armv7'
+
+			# ARMv8
+			elif (( $G_HW_ARCH == 3 )); then
+				arch='arm64'
 
+			# x86_64
+			elif (( $G_HW_ARCH == 10 )); then
+				arch='amd64'
+
+			fi
+
+			local fallback_url="https://github.com/filebrowser/filebrowser/releases/download/v2.17.1/linux-$arch-filebrowser.tar.gz"
+			Download_Install "$(curl -sSfL 'https://api.github.com/repos/filebrowser/filebrowser/releases/latest' | mawk -F\" "/\"browser_download_url\": .*\/linux-$arch-filebrowser\.tar\.gz\"/{print \$4}")" ./filebrowser/
+
+			# Reinstall
+			[[ -d '/opt/filebrowser' ]] && G_EXEC rm -R /opt/filebrowser
+
+			# Install
+			G_EXEC cp -a filebrowser /opt/
+			G_EXEC rm -R filebrowser
+
+			# User
+			[[ -d '/mnt/dietpi_userdata/filebrowser' ]] || G_EXEC mkdir -p /mnt/dietpi_userdata/filebrowser
+			Create_User -G dietpi -d /mnt/dietpi_userdata/filebrowser filebrowser
+
+			# Config if not exist
+			if [[ ! -f '/mnt/dietpi_userdata/filebrowser/filebrowser.db' ]]
+			then
+				G_EXEC cd /mnt/dietpi_userdata/filebrowser/
+				G_EXEC /opt/filebrowser/filebrowser config init
+				G_EXEC /opt/filebrowser/filebrowser config set -a 0.0.0.0 -p 8085 -r /mnt
+				G_EXEC_DESC="Setting up File Browser login user 'dietpi'" G_EXEC /opt/filebrowser/filebrowser users add dietpi $GLOBAL_PW
```

Comment:
`$GLOBAL_PW` should be quoted here to pass special characters and spaces literally.

Reply:
`$GLOBAL_PW` has been quoted now